sanic/response.py

```diff
@@ -1,6 +1,10 @@
 from mimetypes import guess_type
 from os import path
-from ujson import dumps as json_dumps
+
+try:
+    from ujson import dumps as json_dumps
+except ImportError:
+    from json import dumps as json_dumps
```

Comment:
This just a personal preference, by not just just do this? I guess it increase the the amount of changes needed but...

```python
try:
     import ujson as json
except ImportError:
    import json
```